src/OrchardCore/OrchardCore.DisplayManagement/Shapes/PageTitleShapes.cs

```diff
@@ -32,7 +31,7 @@ public IPageTitleBuilder Title
         }
 
         [Shape]
-        public async Task<IHtmlContent> PageTitle(IHtmlHelper Html)
+        public async Task<IHtmlContent> PageTitle()
```

Comment:
this isn't your change @jtkech but is shouldn't need to be using `ShellScope` as the service provider is available on the `Context` when injected

Reply:
@deanmarcussen 

Yes i agree by using `PageTitle(DisplayContext DisplayContext)`.

Anyway, `PageTitleShapes : IShapeAttributeProvider` is a scoped service, why not just injecting the needed services in its ctor. I tried it and it works, but i noticed that the ctor is called twice. I found why, it is resolved through an `IEnumerable<>` in `ShapeAttributeBindingStrategy` and also on demand through the shape binding delegate.

This is because

    public static IServiceCollection AddShapeAttributes<T>(this IServiceCollection services) ...
    {
        services.AddScoped<T>();
        services.AddScoped<IShapeAttributeProvider, T>();

        return services;
    }

In a given scope not good to have multiple instances of the same scoped service, so i think i will do another PR to fix it by using the following

    public static IServiceCollection AddShapeAttributes<T>(this IServiceCollection services) ...
    {
        services.AddScoped<T>();
        services.AddScoped<IShapeAttributeProvider>(sp => sp.GetRequiredService<T>());

        return services;
    }